packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -2134,20 +2134,18 @@ class InteractiveGraph extends React.Component<Props, State> {
         coords = InteractiveGraph.pointsFromNormalized(props, [
             [0.85, 0.5],
             [0.5, 0.5],
-        ]);
+        ]) as [Coord, Coord, Coord];
```

Comment:
Oooo I think no, actually, as the return from pointsFromNormalized is a readonly array. 

This was a necessary type update to support the older graphs so that we can better types for the new graph. This code will be stripped out when we remove the legacy graph, so I wonder if it's better to just leave it as "as" for now.

Reply:
If it's destined to be removed eventually, I think leaving it with `as` is fine for now.